app/controllers/api/open_library_controller.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module Api
```

Comment:
@maxy96 Change the base branch, you already included these changes into another PR # 3